packages/manager/apps/web/client/app/hosting/database/DATABASE_LIST.html

```diff
@@ -105,6 +105,16 @@
                                     data-ng-disabled="!ctrlTabDatabases.canCreateDatabase"
                                 ></button>
                             </li>
+                            <li
+                                data-ng-if="hosting.sqlPriveInfo.nbDataBaseInclude > 0 && hosting.sqlPriveInfo.nbDataBaseActive < hosting.sqlPriveInfo.nbDataBaseInclude"
+                            >
+                                <a
+                                    class="btn btn-link"
+                                    data-translate="hosting_dashboard_service_privatesql_active_long"
+                                    data-ui-sref="app.hosting.database.private-sql-activation"
```

Comment:
Can this extracted to routing ? Or at least to the controller that way the html is less aware of ui-router ?

Reply:
in routing i don't think so, since this template is ng-included it does not work :/

in the controller it might be possible but since there are already other ui-sref in the template it would make sense only if i refactor all the other ui-sref